cosmo/__main__.py

```diff
@@ -73,17 +70,18 @@ def noop(*args, **kwargs):
         if allowed_hosts and device['name'] not in allowed_hosts and device_fqdn not in allowed_hosts:
             continue
 
-        l.info(f"Generating {device_fqdn}")
+        print(f"[INFO] Generating {device_fqdn}")
```

Comment:
I would like to see a better solution for the [INFO] prefix. IMHO this should be at least wrapped into a helper.

Reply:
I have added a small wrapper, not sure there's a need to make something more complex